projects/control-service/projects/base/src/main/java/com/vmware/taurus/security/SecurityConfiguration.java

```diff
@@ -159,6 +168,31 @@ private SecurityFilterChain enableSecurity(
     return http.build();
   }
 
+  @Bean
+  JwtIssuerAuthenticationManagerResolver byIssuer(JwtAuthenticationConverter converter) {
```

Comment:
When authentication is disabled issuer should be ignored (as it is not needed)
If auth is enabled there should be at least one issuer, I think.